querybook/webapp/components/Announcements/Announcements.tsx

```diff
@@ -24,7 +24,12 @@ export const Announcements: React.FunctionComponent = () => {
         dispatch(querybookUIActions.loadDismissedAnnouncements());
     }, []);
 
-    useInterval(loadAnnouncements, 300000);
+    useInterval(() => {
```

Comment:
`useInterval(useCallback(`
since the function passed is in the deps array of useInterval, it will cause useEffect update upon render